src/services/conversation.py

```diff
@@ -19,6 +19,8 @@
 
 logger = get_logger(__name__)
 
+TOKEN_LIMIT = 14_000
```

Comment:
is `14_000` what model approximately can accept at most? Why did you decide for this number?

Reply:
That number indeed is wrong, I will fix it.
https://platform.openai.com/docs/models/gpt-4o